### PR TITLE
Add fake pulse datasets

### PR DESCRIPTION
**Pull request by ylipacbio**

### CODE REVIEW

File: hdf/HDFBaseCallsWriter.cpp [hunk 1]
```diff
@@ -111,28 +113,17 @@ bool HDFBaseCallsWriter::InitializeQVGroups(void) {
...
-                              PacBio::AttributeValues::Common::schemarevision);
-    schemaRevisionAtom.Close();
+bool HDFBaseCallsWriter::_WriteAttributes(void) {
+    // SchemaRevision
+    bool OK = 
+        AddAttribute(basecallsGroup_,
+                     PacBio::AttributeNames::Common::schemarevision,
+                     PacBio::AttributeValues::Common::schemarevision) and
+    // ChangeListID
+        AddAttribute(basecallsGroup_,
```
Comment (pb-cdunn) [inline hdf/HDFBaseCallsWriter.cpp]: Note that this is short-circuited. The 2nd `AddAttribute()` will not be called if the first returns `false`. That might be a change in behavior. Is that ok?

File: hdf/HDFPulseCallsWriter.cpp [hunk 2]
```diff
@@ -106,7 +224,7 @@ std::vector<std::string> HDFPulseCallsWriter::Errors(void) const {
     return retErrors;
 }
 
-HDFPulseCallsWriter::~HDFPulseCallsWriter(void) {
+HDFPulseCallsWriter::~HDFPulseCallsWriter(void) {
```
Comment (pb-cdunn) [inline hdf/HDFPulseCallsWriter.cpp]: Your editor is adding trailing spaces in several locations.

File: hdf/HDFBaseCallsWriter.cpp (hunk 1 above)
Comment (ylipacbio) [inline hdf/HDFBaseCallsWriter.cpp]: It's OK, since it failed anyway.

File: hdf/HDFPulseCallsWriter.cpp (hunk 2 above)
Comment (ylipacbio) [inline hdf/HDFPulseCallsWriter.cpp]: possibly my bad, not the editor.

**Pull request merged**